YAPA/MainWindow.xaml.cs

```diff
@@ -598,7 +586,9 @@ private void Stop_Click(object sender, RoutedEventArgs e)
             else
             {
                 ResetTicking();
+                _isWork = false;
                 PlayMusic();
+                _isWork = true;
```

Comment:
yeah,  I did this change because I found if you're in a promodo time and click stop button twice, it will start to play the workMusic(should be the breakMusic)